core-backend/sandbox/src/env.rs

```diff
@@ -142,6 +142,7 @@ where
         env_builder.add_host_func("env", "gr_wait", funcs::wait);
         env_builder.add_host_func("env", "gr_wake", funcs::wake);
         env_builder.add_host_func("env", "gas", funcs::gas);
+        env_builder.add_host_func("env", "gr_create_program", funcs::create_program);
```

Comment:
We tried to sort syscalls alphabetically.

Reply:
done